### PR TITLE
Increase connection max size

### PR DESCRIPTION
**Pull request by beckmani**

### CODE REVIEW

File: tests/deployment.py [hunk 1]
```diff
@@ -28,7 +28,10 @@ def wait_to_deployment_to_be_ready(deployment_name, name_space, time_out=None):
         resp = client.AppsV1Api().read_namespaced_deployment(name=deployment_name, namespace=name_space)
         total_sleep_time = (datetime.now()-start).total_seconds()
         if not resp.status.unavailable_replicas:
-            print("Total time waiting for deployment {0}: {1} sec".format(deployment_name, total_sleep_time))
+            ready_replicas = resp.status.ready_replicas
```
Comment (y0sher) [inline tests/deployment.py]: I think that's part of an other pr

File: tests/misc.py [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+
+class K8SApiClient(ApiClient):
+
+    def __init__(self):
+        client_config = Configuration()
+        client_config.connection_pool_maxsize = 32
+        super(K8SApiClient, self).__init__(configuration=client_config)
+
+
+class CoreV1ApiClient(client.CoreV1Api):
```
Comment (y0sher) [inline tests/misc.py:13]: we could use a shorter name :)

Comment (beckmani) [reply tests/misc.py:13]: tried to keep the same length as before ;-)

File: tests/deployment.py (hunk 1 above)
Comment (beckmani) [reply tests/deployment.py]: Basically you are right.
But, this change only does not justify being in its own PR

**Review by y0sher — APPROVED** [review]

**Comment by y0sher** [comment]
it would be helpful if you write a few words about why this change is required. (maybe an issue or a comment.)

**Comment by beckmani** [comment]
connection_pool_maxsize = Number of connections to save that can be reused. More than 1 is useful in multithreaded situations.
Whenever there are many connection (HTTP) simultaneously and the pool is small, we may end up with timeout.
The default pool size is: "Number of CPUs" * 5

**Pull request merged**